web_external/views/body/SubmissionView.js

```diff
@@ -1,23 +1,42 @@
-/**
-* View for an individual submission.
-*/
-covalic.views.SubmissionView = covalic.View.extend({
+import _ from 'underscore';
+import { getCurrentUser } from 'girder/auth';
+import { AccessType, DATE_SECOND } from 'girder/constants';
```

Comment:
Unfortunately runtime is the only way for this style of imports. It'll detect bad paths statically, but not bad symbols.